buildSrc/src/main/java/org/elasticsearch/plugin/PluginPropertiesExtension.java

```diff
@@ -0,0 +1,162 @@
+package org.elasticsearch.gradle.plugin;
+
+import org.gradle.api.Project;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.InputFile;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A container for plugin properties that will be written to the plugin descriptor, for easy
+ * manipulation in the gradle DSL.
+ */
+public class PluginPropertiesExtension {
+    
+    @Input
```

Comment:
In groovy, the `Input` annotation adds the get/set capabilities, and is automatically tracked. However, that is for tasks. I'm not sure it actually makes sense (and was erroneous but harmless in the old groovy code). @atorok can you confirm my understanding? If so, I think all these Input annotations should be removed.

Reply:
Thanks for your feedback, I will remove the `Input` annotations.